### PR TITLE
Adding support for the Flux batch scheduler.  

### PR DESCRIPTION
**Pull request by bvanessen**
Updating the Corona CI pipeline to use flux commands.  Adding a Flux launcher script to the
PFE.

Temporarily disabling all of the other pipelines.

### CODE REVIEW

File: .gitlab/corona/pipeline.yml [hunk 1]
```diff
@@ -190,6 +201,9 @@ release allocation:
     # problems if it's not set.
     LBANN_CI_RUN_WEEKLY: ${LBANN_CI_RUN_WEEKLY}
 
+    # Preserve any spack-build-* directories to allow for faster testing
+    GIT_CLEAN_FLAGS: -ffdx -e spack-build-*
+
```
Comment (benson31) [inline .gitlab/corona/pipeline.yml:206]: I don't understand why these are needed? The catch tests run with the build, the relevant logs are stored as artifacts, and the `ci_tests/` directory doesn't use the build tree at all.

Comment (benson31) [reply .gitlab/corona/pipeline.yml:206]: Oh nevermind. This is to accelerate successive pipelines in which one happens to have gotten the same runner (and Spack hashes LBANN the same way). My concern, though, is that, as hashes accumulate, we will just eat up our disk space with stale build directories. We may want to augment this approach with a cron job to clean that waste up (e.g., directories with mtime older than a week or something).

Comment (bvanessen) [reply .gitlab/corona/pipeline.yml:206]: @benson31 I agree that we should have a cron job to clean these directories and spack environments.

**Review by benson31 — APPROVED** [review]
General code looks great. There's some extra debugging output in there still, but it's just Gitlab logging, so that's fine.

Just a note that the Corona unit/integration tests are commented out.

**Review by benson31 — APPROVED** [review]
LGTM

**Pull request merged**